roles/openshift_logging/defaults/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+openshift_logging_curator_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
interesting, i didn't think that would have a value... i guess the default variable was lazy loaded after we had already loaded variables from the correct `var/` file. Excellent!